Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    guard-standardrb (0.1.0)
+    guard-standardrb (0.2.0)
```

Comment:
Did all these dependencies need bumping? Can the PR be achieved without the bump?

Reply:
I think it's when I added the `pry-byebug` that other gems get updated. Might as well update them I think